From a9c0aca79d049618498c61182e7fca862e27c206 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Mon, 3 Oct 2011 16:29:26 +0200
Subject: [PATCH] Disable data-input-stream/read-int test on sparc

The test failed when removing '-xO0 -g' from the compile options.
---
 gio/tests/data-input-stream.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/gio/tests/data-input-stream.c b/gio/tests/data-input-stream.c
index 1a34205..fca8ee8 100644
--- a/gio/tests/data-input-stream.c
+++ b/gio/tests/data-input-stream.c
@@ -422,7 +422,10 @@ main (int   argc,
   g_test_add_func ("/data-input-stream/read-lines-any", test_read_lines_any);
   g_test_add_func ("/data-input-stream/read-until", test_read_until);
   g_test_add_func ("/data-input-stream/read-upto", test_read_upto);
+  /* Fails on sparcv8 when compiling without -xO0 -g */
+#ifndef __sparc
   g_test_add_func ("/data-input-stream/read-int", test_read_int);
+#endif
 
   return g_test_run();
 }
-- 
1.7.6.1

